### DISCUSSION TITLE
kitty window top bar is broken

### DISCUSSION THREAD

**Discussion by LelloPisello** [kovidgoyal/kitty#5936] [Q&A]
hello, I've been using kitty on gnome for a while now, but the top bar does not look like it should on gnome. it is about 16px tall and completely white, with the window title written in the top left. I have tried switching the display server in the configuration to both X11 and wayland but nothing changes. any idea what I'm doing wrong? thanks in advance.

**Comment by kovidgoyal** [comment]
See #3284

If you switch to x11 *correctly* it will work.
